src/game/RandomizationOfShipsPositions.js

```diff
@@ -86,40 +88,49 @@ const positionShipHorizontally = (shipIndex, direction) => {
   if (
     gameBoardArray[position[0]]
       .slice(position[1], position[1] + SHIPS_LIST[shipIndex].size)
-      .some((el) => el.isEmpty === false)
+      .some((el) => el === Object && !el.isEmpty)
```

Comment:
How `eqeqeq` with `Object` constructor is helping here?

Reply:
without Object constructor i had problems that ships have been placed for example half in 3rd row, half in 4th row. I tried to find solution for this problem and it works. Also tried `typeOf Object` but it wasn't helpful